src/components/SubjectInterest.vue

```diff
@@ -3,88 +3,148 @@
+          <label class="custom-input" for="one">have parkinsons</label>
+          <br>
+        </span>
 
-    <br class="visible-md-up">
-    <br class="visible-md-up">
+        <span v-bind:class= "{dim: !(selectedOptionOne === null || selectedOptionOne === 'Two')}">
+          <input   type="radio" id="two" value="Two" v-model="selectedOptionOne">
+          <label class="custom-input" for="two">have another movement disorder</label></label>
+          <br>
+        </span>
 
+        <span v-bind:class= "{dim: !(selectedOptionOne === null || selectedOptionOne === 'Three')}">
+          <input  type="radio" id="three" value="Three" v-model="selectedOptionOne" >
+          <label class="custom-input" for="three">would like to be a control subject</label>
+          <br>
+        </span>
+
+      </form>
 
-    <div class="row" v-if="isInterested">
-      <p class="pageOne col-md-8 offset-md-2 pl-2 pl-md-0"> and I would be willing to try </p>
-      <br class="visible-md-up">
       <br class="visible-md-up">
       <br class="visible-md-up">
+      <p v-if="isInterested" class="col-md-8 offset-md-2 pl-2 pl-md-0"> and I would be willing to try </p>
+      <div class=" col-md-8 offset-md-2" id="selectTwo" v-if="isInterested">
 
-      <select v-focus="isInterested" class="custom-select col-md-6 offset-md-2 customizedSelect" id="selectTwo"
-              v-model="selectedOptionTwo">
-        <option disabled value="">Please select one</option>
-        <!--TODO: Fill in with actual values-->
-        <option> this study </option>
-        <option> parts of this study </option>
-      </select>
-    </div>
+        <div v-bind:class= "{dim: !(selectedOptionTwo === null || selectedOptionTwo === 'One')}">
+          <input type="radio" id="one" value="One" v-model="selectedOptionTwo">
+          <label class="custom-input" for="one"> this study</label>
+          <br>
+        </div>
+
+        <span v-bind:class= "{dim: !(selectedOptionTwo === null || selectedOptionTwo === 'Two')}">
+          <input  type="radio" id="two" value="Two" v-model="selectedOptionTwo">
+          <label class="custom-input" for="two"> parts of this study</label></label>
+          <br>
+        </span>
 
-    <br>
-    <br>
-    <button v-focus="isEligible" :disabled="!isEligible" id="next" v-on:click="clicked"> Next </button>
+      </div>
 
+    </div>
+    <div class="row">
+    <button class="text-center" type="submit" v-if="isEligible" v-focus="isEligible" id="next" v-on:click="clicked"> Next </button>
+    </div>
   </div>
 </template>
 
 <script>
-  import { Focus } from '@/directives/focus.js'
-  export default {
-    name: 'subjectInterest',
-    data () {
-      return {
-        selectedOptionOne: '',
-        selectedOptionTwo: ''
-      }
+import { Focus } from '@/directives/focus.js'
+import _ from 'lodash'
+import Multiselect from 'vue-multiselect'
+
+export default {
+  name: 'subjectInterest',
+  data () {
+    return {
+      selectedOptionOne: null,
```

Comment:
It'd be easier to understand the purpose of these variables if they were named something like selectedOptionForInterest and selectedOptionForWilling